config/initializers/devise.rb

```diff
@@ -235,7 +235,7 @@
   # ==> OmniAuth
   # Add a new OmniAuth provider. Check the wiki for more information on setting
   # up on your models and hooks.
-  config.omniauth :github, ENV['GITHUB_ID'], ENV['GITHUB_SECRET'], scope: 'user:email'
+  config.omniauth :github, ENV['GITHUB_ID'], ENV['GITHUB_SECRET']
```

Comment:
It's just extra if we aren't collecting email. The scope will default to basic authentication, which includes uid and email.